{{cookiecutter.repo_name}}/{{cookiecutter.project_name}}/tests/routes/test_example_routes.py

```diff
@@ -71,7 +71,7 @@ def test_search(self):
 
         with self.graph.app.test_request_context():
             assert_that(response.status_code, is_(equal_to(200)))
-            data = loads(response.data)
+            data = loads(response.data.decode("utf-8"))
```

Comment:
`response.data` are `bytes` in Python3; need to convert to string to pass to `loads`.